.gitignore

```diff
@@ -1 +1 @@
-
+.idea/
```

Comment:
'should be part of your local .gitignore, not project's one, btw :)

Reply:
yes, we currently add IDE config to global ignore list. You can find it in the [documentation](https://app.gitbook.com/@cospirit-connect/s/guide-de-demarrage/installation-des-projets/prerequis/configuration-globale-git).